rust-executor/src/holochain_service/mod.rs

```diff
@@ -313,7 +329,7 @@ impl HolochainService {
             _ => unreachable!(),
         };
 
-        inteface.add_agent_infos(agent_infos_from_str(COASYS_BOOTSTRAP_AGENT_INFO).expect("Couldn't deserialize hard-wired AgentInfo")).await?;
+        // inteface.add_agent_infos(agent_infos_from_str(COASYS_BOOTSTRAP_AGENT_INFO).expect("Couldn't deserialize hard-wired AgentInfo")).await?;
```

Comment:
I remember you talked about this, but I don't remember why you had to deactivate this line, @fayeed. We will need to put it back in for production, right? Was it that we had to recreate the bootstrap agent with the same new Holochain version?